sync/sync_test.go

```diff
@@ -22,6 +22,7 @@ func TestSyncBlocks(t *testing.T) {
 			require.NoError(t, err)
 
 			height := int(headBlock.Number)
+			assert.True(t, height > 0)
```

Comment:
we should assert the height to be 2 and if this fails we should increase sleep time

Reply:
done